cms/static/js/views/pages/container.js

```diff
@@ -40,11 +40,19 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
             initialize: function(options) {
                 BasePage.prototype.initialize.call(this, options);
                 this.viewClass = options.viewClass || this.defaultViewClass;
+                this.isUnitPage = this.options.isUnitPage;
+                this.isSplitTest = (this.model.attributes.category === 'split_test');
```

Comment:
I think we want to generalize this so that it will work with other xblocks that have children (like the conditional xblock). I think that we want to show the editor whenever we have a vertical (whenever we are rendering the container page). What happens if you just always add the access editor instead of making it conditional?